svir/dialogs/upload_dialog.py

```diff
@@ -137,11 +137,15 @@ def _update_layer_style(self):
             os.close(fd)
             with open(fname, 'w') as f:
                 f.write(sld)
-            os.system('tidy -xml -i %s' % fname)
+            # os.system('tidy -xml -i %s' % fname)
         headers = {'content-type': 'application/vnd.ogc.sld+xml'}
+        resp = self.session.post(
```

Comment:
Please add a comment here, clarifying why we are doing both a post and a put. For instance: `NOTE: to save the style, we actually must do both the post and the put in this sequence, otherwise it does not work. We still haven't found any better way to obtain the same result.`